### PR TITLE
[TestSwiftBridgedMetaType] Modify and enable on Linux.

### PR DESCRIPTION
**Pull request by dcci**
<rdar://problem/55632370>

### CODE REVIEW

**Comment by dcci** [comment]
@adrian-prantl @jimingham please take a look.

**Comment by dcci** [comment]
@swift-ci test and merge

File: packages/Python/lldbsuite/test/lang/swift/bridged_metatype/Makefile [hunk 1]
```diff
@@ -1,6 +1,4 @@
 LEVEL = ../../../make
-
 SWIFT_SOURCES := main.swift
 SWIFT_OBJC_INTEROP := 1
```
Comment (adrian-prantl) [inline packages/Python/lldbsuite/test/lang/swift/bridged_metatype/Makefile:3]: This seems incorrect, as it works with Foundation on Linux, too.

File: packages/Python/lldbsuite/test/lang/swift/bridged_metatype/TestSwiftBridgedMetatype.py [hunk 2]
```diff
@@ -37,10 +25,7 @@ def test_swift_bridged_metatype(self):
...
         var_k = self.frame().FindVariable("k")
-        lldbutil.check_variable(self, var_k, False, "NSString")
-
-if __name__ == '__main__':
-    import atexit
-    lldb.SBDebugger.Initialize()
-    atexit.register(lldb.SBDebugger.Terminate)
-    unittest2.main()
+        if sys.platform.startswith("linux"):
+            lldbutil.check_variable(self, var_k, False, "Foundation.NSString")
```
Comment (adrian-prantl) [inline packages/Python/lldbsuite/test/lang/swift/bridged_metatype/TestSwiftBridgedMetatype.py:29]: Do we understand why this is printed differently?

Comment (dcci) [reply packages/Python/lldbsuite/test/lang/swift/bridged_metatype/TestSwiftBridgedMetatype.py:29]: No, do you have ideas?

**Comment by dcci** [comment]
@swift-ci test macos platform

**Pull request merged**